pubsub/.flake8

```diff
@@ -4,3 +4,9 @@ exclude =
   .git,
   *.pyc,
   conf.py
+
+ignore =
```

Comment:
I will re-enable it and go back and check and see what it complained about. If it's a PEP 8 violation, then I will change it and put the lint back. I am 95% sure it was not.